python_ta/reporters/node_printers.py

```diff
@@ -153,6 +205,328 @@ def render_blank_line(line):
     yield (line + 1, slice(None, None), LineType.ERROR, " " * 28)
 
 
+def render_pep8_errors_e101(msg, _node, source_lines=None):
+    """Render a PEP8 indentation contains mixed spaces and tabs message."""
+    line = msg.line - 1
+    curr_idx = 0
+    while source_lines[line][curr_idx].isspace():
+        curr_idx += 1
+    yield from render_context(line - 1, line + 1, source_lines)
+    yield (msg.line, slice(0, curr_idx), LineType.ERROR, source_lines[line])
```

Comment:
In general don't mix `line` and `msg.line` in the `yield` statements. Use the variable `line` consistently throughout (as we do in `render_generic`).